lib/ansible/modules/packaging/os/pkgutil.py

```diff
@@ -27,30 +27,43 @@
     - Pkgutil is an advanced packaging system, which resolves dependency on installation.
       It is designed for CSW packages.
 version_added: "1.3"
-author: "Alexander Winkler (@dermute)"
+author:
+    - "Alexander Winkler (@dermute)"
+    - "David Ponessa (github.com/scathatheworm)"
 options:
   name:
     description:
       - Package name, e.g. (C(CSWnrpe))
+      - When using state=latest, this can be '*', which updates all installed packages managed by pkgutil.
+      - Multiple packages can be specified using a comma as a separator.
     required: true
+    aliases: [ 'pkg' ]
   site:
     description:
       - Specifies the repository path to install the package from.
       - Its global definition is done in C(/etc/opt/csw/pkgutil.conf).
     required: false
   state:
     description:
-      - Whether to install (C(present)), or remove (C(absent)) a package.
-      - The upgrade (C(latest)) operation will update/install the package to the latest version available.
-      - "Note: The module has a limitation that (C(latest)) only works for one package, not lists of them."
+      - Whether to install (C(present)), or remove (C(absent)) packages.
+      - The upgrade (C(latest)) operation will update/install the packages to the latest version available.
     required: true
     choices: ["present", "absent", "latest"]
   update_catalog:
     description:
       - If you want to refresh your catalog from the mirror, set this to (C(yes)).
     required: false
-    default: False
+    choices: [ "yes", "no" ]
```

Comment:
There should be no spaces at the beginning and end of the square brackets.

Reply:
actually choices should be removed and add `type: bool` instead as the [docs](http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html) says